enos/enos.vars.hcl

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+artifactory_username = "<your email address>"
+artifactory_token    = "<your ARTIFACTORY_TOKEN from above>"
+product_version      = "1.8.9"                     # starting version
+upgrade_version      = "1.9.4"                     # version to upgrade to
+download_binary_path = "/home/foo/Downloads/nomad" # directory on your machine to download binaries
+nomad_license        = "<your Nomad Enterprise license, currently always required>"
+consul_license       = "<your Consul Enterprise license, currently always required>"
+aws_region           = "us-east-1"
```

Comment:
Should this really exist here in the repo? We're going to have to overwrite it during GHA runs, leaving a dirty repo state.